File: templates/Uwp/_comp/MVVMLight/Service.IdentityForcedLogin._VB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Private Sub New()
'^^
'{[{
            Register(Of LogInViewModel, LogInPage)()
'}]}
        End Sub

'{[{
        Public ReadOnly Property LogInViewModel As LogInViewModel
            Get
                Return SimpleIoc.[Default].GetInstance(Of LogInViewModel)()
            End Get
        End Property
'}]}
    End Class
End Namespace

File: templates/Uwp/_comp/MVVMLight/Service.IdentityForcedLogin/ViewModels/ViewModelLocator_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        private ViewModelLocator()
        {
            //^^
            //{[{
            Register<LogInViewModel, LogInPage>();
            //}]}
        }

        //{[{
        public LogInViewModel LogInViewModel => SimpleIoc.Default.GetInstance<LogInViewModel>();
        //}]}
    }
}
